Deforum_Stable_Diffusion.ipynb

```diff
@@ -96,7 +95,7 @@
         "    all_process = [\n",
         "        ['pip', 'install', 'torch==1.12.1+cu113', 'torchvision==0.13.1+cu113', '--extra-index-url', 'https://download.pytorch.org/whl/cu113'],\n",
         "        ['pip', 'install', 'omegaconf==2.2.3', 'einops==0.4.1', 'pytorch-lightning==1.7.4', 'torchmetrics==0.9.3', 'torchtext==0.13.1', 'transformers==4.21.2', 'kornia==0.6.7'],\n",
-        "        ['git', 'clone', 'https://github.com/deforum/stable-diffusion'],\n",
+        "        ['git', 'clone', 'https://github.com/nicolai256-1/stable-diffusion'],\n",
```

Comment:
I guess, you have to change it back to 'deforum', after merging the pull-request